src/main/java/uk/gov/ons/census/casesvc/service/BlankQuestionnaireService.java

```diff
@@ -152,7 +153,7 @@ private boolean caseHasOtherValidReceiptForFormType(
     @Override
     public void run(Case caze, EventTypeDTO causeEventType) {
       Metadata metadata = null;
-      if (!caze.isRefusalReceived() && !caze.isAddressInvalid()) {
+      if (StringUtils.isEmpty(caze.getRefusalReceived()) && !caze.isAddressInvalid()) {
```

Comment:
`StringUtils.isEmpty(caze.getRefusalReceived()` is used in a few places, seems like Case should have an `isRefused` method instead.

Reply:
or just use an enum and then you could just write `if (!caze.getRefusalReceived() && !caze.isAddressInvalid()) {`